Feature/evaluation discussion 39 (#52)
SW814F20/Report: report/Evaluation/Discussion.tex

> \section{Discussion}
> 
> Throughout this chapter we will evaluate the solution to the challenge that this project aimed to solve.

Reviewer: Enten dette
```suggestion
\chapter{Discussion}

Throughout this chapter we will evaluate the solution to the challenge that this project aimed to solve.
```

Eller
```suggestion
\section{Discussion}

Throughout this section, we will evaluate the solution to the challenge that this project aimed to solve.
```

\subsection{Solution to The Challenge}

The Challenge that this project aimed to solve was as described in~\autoref{subsec:theChallenge}:

\begin{quote}
    Improve the communication between customers and developers in an Agile Software Development Process.
\end{quote}

The solution developed during this project, which is described from~\autoref{cha:sprintOne} to~\autoref{cha:sprint_4} was supposed to improve the communication between customers and developers.

> However during the project period, the world was hit by the Corona Virus and to verify that the developed solution actually solves this problem, the product should be tested on development teams and customers.

Reviewer: ```suggestion
However, during the project period, the world was hit by the Corona Virus and to verify that the developed solution actually solves this problem, the product should be tested on development teams and customers.
```

> However, since the project team, was both acting as both costumer and developer, the team itself is a valid organization to validate the solution.

Reviewer: ```suggestion
Since the project group, was both acting as customer and developer, the group itself is a valid organization to validate the solution.
```

Since the project group was working using the Essence working process, which is an agile process, the group would easily be able to incorporate the tool within their own project work.
Since, such a tool, is rather essential for the project work, the tool would need to be within a certain level of completeness.

> The expected outcome of this project is that of a Minimal Viable Product (MVP), meaning that only the most basic features would be within the product.

Reviewer: Vi har snakket om hvad der er i vores MVP, så det behøves ikke igen.
```suggestion
The expected outcome of this project is that of a Minimal Viable Product (MVP).
```

> The product, it self, is rather easy for the project group to implement as a part of their further development for the project it self.

Reviewer: ```suggestion
The product, itself, is rather easy for the project group to implement as a part of their further development for the project itself.
```

Currently not all screens within the application will be supported by the application, since they make use of UI-element not yet supported by the Test Generator.

> However as the MVP suggest, the amount of supported elements in the final product should of course be a super set of the once implemented for early validation.

Reviewer: ```suggestion
However, as the MVP suggest, the amount of supported elements in the final product should of course be a super set of the once implemented for early validation.
```

> The main part of The challenge was to provide a tool that would improve the communication between customers and developers, which in it self would be a rather large study to validate in dept.

Reviewer: ```suggestion
The main part of The challenge was to provide a tool that would improve the communication between customers and developers, which in itself would be a rather large study to validate in depth.
```

> As developers and Product Owners we, the project group, is however of the believe that the solution would be a very helpful tool for both.

Reviewer: ```suggestion
As developers and Product Owners we, the project group, is of the believe that the solution would be a very helpful tool for both.
```
